docs/Developers/Git_Basics.rst

```diff
@@ -168,7 +168,7 @@ parts. They should have a newline between them.
 .. seealso::
 
   For more information about writing commit messages, check this
-  `link <http://coala.io/commit>`_.
+  `link <http://coala.io/commit>`__.
```

Comment:
Travis build is passed now.  The intention behind adding a double underscore at the end of hyperlinks was to resolve  (WARNING: Duplicate explicit target name: "link") mentioned error.  Only way out for resolution was to  change the target name or  add double underscore.
Earlier it was failing because by mistake I added triple underscore.